java/execution/impl/src/com/intellij/execution/scratch/JavaScratchCompilationSupport.java

```diff
@@ -153,7 +153,7 @@ else if (isPublic(aClass)) {
         options.add(langLevel);
         options.add("-target");
         options.add(langLevel);
-        if (sdkVersion.isAtLeast(JavaSdkVersion.JDK_11)) {
+        if (sdkVersion.isAtLeast(JavaSdkVersion.JDK_12)) {
```

Comment:
I think this line is better to be changed into this one:

`if (sdkVersion.getMaxLanguageLevel().isPreview()) {`